04_diy_neural_network/README.md

```diff
@@ -0,0 +1,37 @@
+# DIY Neural Network
+
+## Session A: Data Collection
+
+### Objectives:
+* Understand the full story of building a ML model for classification or regression.
+* Understand how data is formatted and downloaded including CSV and JSON.
+* Consider how to frame the problem and collect data.
+    * Understand critical questions to ask (e.g. Who is this for? What’s the context?)
+    * Understand the questions to ask about sourcing and collecting data.
+    * Learn how to prepare a data set, including how to normalize and properly format it.
+* Diagram the components of a two layer "vanilla" neural network.
+
+### Related Projects
+* [Gender Shades: How well do IBM, Microsoft, and Face++ AI services guess the gender of a face?](http://gendershades.org/) by Joy Buolamwini and Timnit Gebru
+* [Feminist Data Set](https://carolinesinders.com/feminist-data-set/) by Caroline Sinders
+
```

Comment:
This project, live until Friday, September 27, 2019, and it’s accompanying paper (which I suggest adding below) is a timely follow-up to the students' investigations into ImageNet and COCO in the previous weeks. It's critical project, especially in conversation with Fei-Fei Li's opinion piece also posted on the syllabus for this week.
```suggestion
*[ImageNet Roulette](https://imagenet-roulette.paglen.com) by Kate Crawford and Trevor Paglen with Leif Ryge
```

Reply:
Oh wow! On repeated spins, ImageNet Roulette labeled me a first offender, mediatrix, whiteface, lass, psycholinguist, and face.